src/components/Header/Header.scss

```diff
@@ -75,6 +75,12 @@
     .responsive-header {
         display: none;
     }
+    .nav-menu-mobile {
+        display: none;
+    }
+    .dropdown-menu {
+        display: none;
```

Comment:
Kad stavis `display: none`  animacija nece raditi, jer ne mozes da animiras display none.

Reply:
Zato se ovaj div ne animira i izgleda cudno nego ti se samo overlay animira unutar njega

Reply:
Obrisao sam display none, sada postoji i na velicini vecoj od 1360px, gurnuo je u header ceo mobile-nav-menu, probacu da ispravim..

Reply:
E sada, ovo je zapravo poslednji komentar nego se vracam, sve sam resio radi dobro, nema bagova, ostao je jos ovaj problem, ostaje mi ceo meni i na velicini gde ne treba da bude a to je sve vise od 1360px.